R/bundles.R

```diff
@@ -13,31 +15,13 @@
 #' available_bundles()
 #' available_bundles("0.1")
 available_bundles <- function(version = "latest") {
-  raw_dir <- .get_raw_dir(version = version)
-  directories <- list.dirs(raw_dir, full.names = TRUE)
-  domain_directories <- directories[directories != raw_dir]
-
-  purrr::map_dfr(domain_directories, .build_concepts_from_directory) |>
-    mutate(version = version)
-}
+  raw_dir <- omopbundles:::get_raw_dir(version = version)
```

Comment:
```suggestion
  raw_dir <- get_raw_dir(version = version)
```

No need for `:::` as we're in the same package?